.github/workflows/pipeline.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       - run: python -m pip install -e .
       - run: black --check --diff sydent/ tests/ matrix_is_test/ setup.py
       - run: flake8 sydent/ tests/ matrix_is_test/ setup.py
+      - run: isort --check-only --diff sydent/ tests/ matrix_is_test/ setup.py
```

Comment:
Should probably add `--profile black` in here?

Reply:
Actually, let's throw the profile configuration in a `[tool.isort]` stanza in `pyproject.toml` so it gets picked up in local runs, too. See https://pycqa.github.io/isort/docs/configuration/config_files/#pyprojecttoml-preferred-format for more info.

Reply:
will do!